net/znc/Makefile

```diff
@@ -23,6 +23,8 @@ PKG_LICENSE_FILES:=LICENSE
 PKG_USE_MIPS16:=0
 PKG_BUILD_PARALLEL:=1
 
+PKG_CONFIG_DEPENDS := \
+	CONFIG_ZNC_ICU
```

Comment:
I originally had an option for SSL too, as all other packages seem to disable it by default but I took it out. Noted.